README.md

```diff
@@ -208,6 +176,23 @@ func main() {
 
 ```
 
+### Configuring the Sender
+
+```go
+import (
+	wavefront "github.com/wavefronthq/wavefront-sdk-go/senders"
+)
+
+func main() {
+    // Send batches of 20,000. Default is 10,000. Flush every 5 seconds. Default is 1 second.
```

Comment:
Let's put the comment about each option down below, on the same line where the override is occurring. Let's also give the example of overriding the Metrics and Traces ports.

Reply:
Done.